app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -61,9 +59,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             @Override
             public void onRefresh() {
                 DataDownload download = new DataDownload();
-                dbSingleton.clearDatabase(DbContract.Microlocation.TABLE_NAME);
```

Comment:
I go through the code,in api/processor files ...MicrolocationListResponseProcessor..and other files ,....after every success you are doing 
" dbSingleton.clearDatabase(DbContract.Sessions.TABLE_NAME);
                dbSingleton.insertQueries(queries);"
So,it will clear the old data and insert the new data if network allows